### DISCUSSION TITLE
Is there a way to set default options like always use dark mode and always sort by date added?

### DISCUSSION THREAD

**Discussion by davin900** [VueTorrent/VueTorrent#653] [Q&A] [answered]
I love VueTorrent! Thank you @WDaan for this.

Just curious if there's an option somewhere to always have dark mode on? Even for new sessions? Same for sorting the torrents by date added?

I often check Qbit in incognito/private browser windows so it always shows me the light mode first and is sorted by priority, which is not my preference.

Just a small thing I was curious about.

**Comment by WDaan** [comment]
Unfortunately, no, not at this point in time. The settings are stored in local storage... so every way of accessing the browser that clears this upon close, makes you lose the settings.

**Reply by davin900** [reply]
Okay thanks for the quick response!

Would it be possible someday to have server-side settings that are persistent? Happy to create a request under Issues if that's helpful.

**Reply by WDaan** [reply, marked as answer]
For now this is just a WebUI, so it can only work with the browser. There's no real way to store anything elsewhere without modifying the source qbittorent AFAIK.

**Reply by afwhite** [reply]
Good morning,

I was looking for this issue exactly as I am in the same boat as OP, using multiple browsers/devices and having to reset my layout, etc on each. Your answer is what I suspected was the case. I'm coming to Vue from Flood so thank you for all the work you do on this!
